AlmesCore/README.md

```diff
@@ -0,0 +1,9 @@
+# Redirects for the Almes Core Metadata Schema
+
+## Maintainers:
+- GitHub ID: @AlmesCore
+- GitHub ID: @Filipi-Soares
+- GitHub ID: @kellyrb
+
+### Almes Core Management Group: <a href="https://orcid.org/0000-0002-0674-7960"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Filipi Miranda Soares</a>, <a href="https://orcid.org/0000-0002-2744-6104"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Fernando Elias Corrêa</a>, <a href="https://orcid.org/0000-0002-4801-2225"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Dilvan de Abreu Moreira</a>, <a href="https://orcid.org/0000-0003-4177-1322"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Debora Pignatari Drucker</a>, <a href="https://orcid.org/0000-0001-6218-6849"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Kelly Rosa Braghetto</a>, <a href="https://orcid.org/0000-0003-1810-1742"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Alexandre Cláudio Botazzo Delbem</a>, <a href="https://orcid.org/0000-0001-7432-7653"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luís Ferreira Pires</a>, <a href="https://orcid.org/0000-0002-1164-1351"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luiz Olavo Bonino da Silva Santos</a>, and <a href="https://orcid.org/0000-0003-2283-1123"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Antonio Mauro Saraiva</a>.
```

Comment:
That looks challenging to edit.  I'd at least drop the list down into a paragraph rather than it being part of the header.  Whatever you'd like.

```suggestion
### Almes Core Management Group:

<a href="https://orcid.org/0000-0002-0674-7960"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Filipi Miranda Soares</a>, <a href="https://orcid.org/0000-0002-2744-6104"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Fernando Elias Corrêa</a>, <a href="https://orcid.org/0000-0002-4801-2225"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Dilvan de Abreu Moreira</a>, <a href="https://orcid.org/0000-0003-4177-1322"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Debora Pignatari Drucker</a>, <a href="https://orcid.org/0000-0001-6218-6849"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Kelly Rosa Braghetto</a>, <a href="https://orcid.org/0000-0003-1810-1742"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Alexandre Cláudio Botazzo Delbem</a>, <a href="https://orcid.org/0000-0001-7432-7653"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luís Ferreira Pires</a>, <a href="https://orcid.org/0000-0002-1164-1351"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luiz Olavo Bonino da Silva Santos</a>, and <a href="https://orcid.org/0000-0003-2283-1123"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Antonio Mauro Saraiva</a>.
```

Reply:
Toward easier future editing... with the same visual rendering as @davidlehn's suggestion
```suggestion
### Almes Core Management Group:

<a href="https://orcid.org/0000-0002-0674-7960"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Filipi Miranda Soares</a>, 
<a href="https://orcid.org/0000-0002-2744-6104"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Fernando Elias Corrêa</a>, 
<a href="https://orcid.org/0000-0002-4801-2225"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Dilvan de Abreu Moreira</a>, 
<a href="https://orcid.org/0000-0003-4177-1322"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Debora Pignatari Drucker</a>, 
<a href="https://orcid.org/0000-0001-6218-6849"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Kelly Rosa Braghetto</a>, 
<a href="https://orcid.org/0000-0003-1810-1742"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Alexandre Cláudio Botazzo Delbem</a>, 
<a href="https://orcid.org/0000-0001-7432-7653"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luís Ferreira Pires</a>, 
<a href="https://orcid.org/0000-0002-1164-1351"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Luiz Olavo Bonino da Silva Santos</a>, and 
<a href="https://orcid.org/0000-0003-2283-1123"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon"> Antonio Mauro Saraiva</a>.
```

Reply:
@TallTed @davidlehn Thanks for the suggestion, just updated the code.